streaming/stream_blob.cc

```diff
@@ -656,11 +656,12 @@ future<stream_files_response> tablet_stream_files_handler(replica::database& db,
     blogger.debug("stream_sstables[{}] Started sending sstable_nr={} files_nr={} files={} range={}",
             req.ops_id, sstables.size(), files.size(), files, req.range);
```

Comment:
nit: files.size() is actually logged first here (in debug level, along with the list of files)

Reply:
Orthogonal to the PR though